examples/openapi3/reverseproxy_aiohttp/README.rst

```diff
@@ -0,0 +1,58 @@
+=====================
+Reverse Proxy Example
+=====================
+
+This example demonstrates how to run a connexion application behind a path-altering reverse proxy.
+
+You can either set the path in your app, or set the ``X-Forwarded-Path`` header.
+
+Running:
+
+.. code-block:: bash
+
+    $ sudo pip3 install --upgrade connexion[swagger-ui]  # install Connexion from PyPI
```

Comment:
Better to specify extra dependency:
`sudo pip3 install --upgrade connexion[swagger-ui] aiohttp-remotes`

Reply:
good catch! oops